### ISSUE TITLE
Create Landsat Dataset

### ISSUE THREAD

**Issue by slavakurilyak** [produvia/mapo#3]
# Goal
As a developer, I want to use [landsat-util ](https://github.com/developmentseed/landsat-util) library to create a Landsat dataset, so that I can generate imagery dataset.

# Consider
* Downloading landsat dataset via [Google Cloud ](https://cloud.google.com/storage/docs/public-datasets/landsat) (2016) or [ESA ](http://www.esa.int/Our_Activities/Observing_the_Earth/New_Landsat_data_just_a_few_clicks_away) (2013) re: 1972-2011 imagery

**Comment by ucalyptus** [comment]
Hi @slavakurilyak  I think we can do this without any use of landsat-util.
Consider Google Earth Engine .
[Example](http://ucalyptus.github.io/EarthEngine-Deep-Learning)

**Comment by slavakurilyak** [comment]
@ucalyptus I agree. Using [Google Earth Engine](https://earthengine.google.com/), we can use [geometric operations](https://developers.google.com/earth-engine/geometric_operations) to create buffers, centroid, or bounding boxes for mineral occurrence coordinates.

For example, here are the coordinates (38°48′N, 120°53′W) for the California Gold Rush. We can define a bounding box of 3x3 satellite pixels or a buffer of 1 meter.
